pulpcore/pulpcore/app/viewsets/task.py

```diff
@@ -43,6 +45,14 @@ def cancel(self, request, pk=None):
         return Response(status=status.HTTP_204_NO_CONTENT)
 
 
+class CoreUpdateTaskViewSet(TaskViewSet):
+
+    endpoint_name = 'core/updates'
```

Comment:
includes namespace which allows plugins to use the same name.

Reply:
The idea is that all tasks that are deployed by pulpcore will be listed at v3/tasks/core/, but CoreTask is just abstract.
Updates to Repositories, Importers, Publishers will deploy CoreUpdateTask, listed at v3/tasks/core/updates/
Deletes of the same will deploy CoreDeleteTask, listed at v3/tasks/core/deletes/.

Reply:
Ah, I see, cool